locale/ja/site.json

```
@@ -73,7 +73,7 @@
       "text": "リリースファイルのための SHASUM 署名"
     },
     "install-on-linux": {
-      "text": "Installing Node.js via binary archive"
+      "text": "書庫化バイナリから Node.js をインストールする"
```

書庫化って聞きなれないですね。。「アーカイブ」でいいと思いました。
```suggestion
      "text": "バイナリアーカイブから Node.js をインストールする"
```

---

確かに，連語としての“binary archive”の訳語は「バイナリアーカイブ」が適切ですね……（[dpkgの翻訳](https://github.com/oerdnj/dpkg/blob/master/po/ja.po#L5122)を見ても）。
調査不足で申し訳ないです，「バイナリアーカイブ」に修正して下さい。

---

書庫化バイナリから => アーカイブから

---

なるほど。アーカイブだけでも良さそうですね。